includes/polling/os/ict-mps.inc.php

```diff
@@ -0,0 +1,7 @@
+<?php
+
+//SNMPv2-SMI::enterprises.39145.13.1.0 = STRING: "Modular Power Series 48V"
+//SNMPv2-SMI::enterprises.39145.13.4.0 = STRING: "1.04"
+
+$hardware = trim(snmp_get($device, '1.3.6.1.4.1.39145.13.1.0', '-OQv', '', ''), '" ');
```

Comment:
As you have MIBs for these devices can you not use named OIDs here? Also needs to be snmp_multi_get.

Reply:
Moved to snmp_get_multi.